src/sentry/integrations/github/issues.py

```diff
@@ -114,8 +116,7 @@ def get_link_issue_config(self, group, **kwargs):
         else:
             repo_choices = [(repo['identifier'], repo['name']) for repo in repos]
 
-        params = kwargs.get('params', {})
-        default_repo = params.get('repo', repo_choices[0][0])
+        default_repo = self.get_project_defaults(group.project_id).get('repo') or repo_choices[0][0]
```

Comment:
Do we still want that `repo_choices[0][0]`?

Reply:
@MeredithAnya does github work in a similar way at all to jira, using the params to signal a different state (i.e. what fields should be displayed?)?

Reply:
yeah if they change the repo it will come in the params and we will want to use that, also yes to keeping repo_choices